web3/contracts.go

```diff
@@ -198,6 +198,21 @@ func (c *Contracts) CurrentBlock() uint64 {
 	return c.currentBlock
 }
 
+// SupportBlobTxs returns true if the ProcessRegistry contract supports blobs
+// transactions.
+func (c *Contracts) SupportBlobTxs() bool {
```

Comment:
Implemented like this means that on every state-transition we'll be checking the onchain contract to read a boolean which cannot change (is set on deploy time). 

This overhead is unnecessary, instead do: on start perform the check, store a local boolean, and just use it.